packages/cli/lib/prompts/developerTestAccountNamePrompt.js

```diff
@@ -0,0 +1,29 @@
+const { promptUser } = require('./promptUtils');
+const { i18n } = require('../lang');
+const { accountNameExistsInConfig } = require('@hubspot/local-dev-lib/config');
+
+const i18nKey = 'cli.lib.prompts.developerTestAccountPrompt';
+
+const developerTestAccountNamePrompt = currentPortalCount => {
```

Comment:
Not a blocker, but I wonder if we could DRY things up by making the `accountNamePrompt` abstract enough to support this and also the sandboxNamePrompt use cases.

Reply:
Will be taken care of here: https://git.hubteam.com/HubSpot/sandboxes-ui/issues/2309